src/main/java/session_17_equals_hashcode_generics/challenge/challenge_10/AppInformation.java

```diff
@@ -0,0 +1,40 @@
+package session_17_equals_hashcode_generics.challenge.challenge_10;
+
+public class AppInformation {
+    private String name;
```

Comment:
Please consider leaving an empty row between the class declaration and the first variable/method from the class.